src/app/components/internal/SideNav/_main-nav.scss

```diff
@@ -16,6 +16,13 @@
       outline: 0;
     }
 
+    a:focus{
```

Comment:
Very nit-picky 🙇, but great to have a space before `{` to keep our codebase look coherent.

Reply:
Added the space! Thanks for pointing that out/teaching me that Akira!